spec/spec_helper.rb

```diff
@@ -22,5 +22,21 @@
   config.shared_context_metadata_behavior = :apply_to_host_groups
 
   Capybara.default_max_wait_time = 5
-  Webdrivers::Chromedriver.required_version = '2.46'
-end
\ No newline at end of file
+
+  Webdrivers::Chromedriver.update
+
+  Capybara.register_driver :chrome do |app|
+    Capybara::Selenium::Driver.new(app, browser: :chrome)
+  end
+
+  Capybara.register_driver :headless_chrome do |app|
+    Capybara::Selenium::Driver.new app, browser: :chrome,
+                                   options: Selenium::WebDriver::Chrome::Options.new(args: %w[headless disable-gpu])
+  end
+
+  Capybara.default_driver = :headless_chrome
+  Capybara.javascript_driver = :headless_chrome
+
+    # Capybara.default_driver = :chrome
```

Comment:
Remove the code if it is not necessary

Reply:
@govindkrjoshi 
By default, we are using headless chrome.
But while writing tests, if someone wants to see how test execution is going on they can enable Capybara.default_driver= :chrome and disable code for headless chrome.
It will also help if someone wants to give a  demo for the scripts to another person.


That's why I have commented it.